src/ORM/RulesChecker.php

```diff
@@ -86,9 +89,12 @@ public function isUnique(array $fields, array|string|null $message = null): Rule
      * $rules->add($rules->existsIn('site_id', new SitesTable(), 'Invalid Site'));
      * ```
      *
-     * Available $options are error 'message' and 'allowNullableNulls' flag.
-     * 'message' sets a custom error message.
-     * Set 'allowNullableNulls' to true to accept composite foreign keys where one or more nullable columns are null.
+     * ### Options
+     *
+     * - `allowMultipleNulls` Allows any field to have multiple null values. Defaults to false.
```

Comment:
Before it mentioned __nullable__ null, not it replaces this with __multiple__ nulls?